node-manager/src/node.rs

```diff
@@ -365,6 +367,114 @@ impl Node {
         Ok(invoice)
     }
 
+    pub fn get_invoice(&self, invoice_str: String) -> Result<MutinyInvoice, MutinyError> {
+        let invoice = Invoice::from_str(&invoice_str).map_err(Into::<MutinyError>::into)?;
```

Comment:
Reposting previous comment, but I think this function should take in an `Invoice` instead of `String`, let the parsing be handled on the `wasm` bound functions.

Reply:
fixed here too